var/spack/repos/jcsda-emc-bundles/packages/jedi-base-env/package.py

```diff
@@ -9,8 +9,8 @@
 class JediBaseEnv(BundlePackage):
     """Basic development environment for JEDI applications"""
 
-    homepage = "https://github.com/noaa-emc/spack-stack"
-    git = "https://github.com/noaa-emc/spack-stack.git"
+    homepage = "https://github.com/jcsda/spack-stack"
+    git = "https://github.com/jcsda/spack-stack.git"
```

Comment:
Does this definition of `git` do anything, since this is just a metapackage?

Reply:
No, it's just sitting there. Doesn't hurt though.